### ISSUE TITLE
Tool doesn't download all data - zip is missing

### ISSUE THREAD

**Issue by manycoding** [Kaggle/kaggle-api#268]
Colab environment, Kaggle API 1.5.4

`kaggle competitions download -c plant-pathology-2020-fgvc7`
downloads all but `plant-pathology-2020-fgvc7.zip`

Which is useless. I expect it to download all competition's data, the same as in input directory in a Kaggle kernel.

`kaggle competitions files -c  plant-pathology-2020-fgvc7`
```
Warning: Looks like you're using an outdated API Version, please consider updating (server 1.5.6 / client 1.5.4)
name                    size  creationDate         
---------------------  -----  -------------------  
images/Test_1010.jpg   283KB  2020-03-09 20:57:51  
images/Test_1002.jpg   214KB  2020-03-09 20:57:51  
images/Test_1013.jpg   170KB  2020-03-09 20:57:51  
images/Test_1001.jpg   226KB  2020-03-09 20:57:51  
images/Test_1011.jpg   290KB  2020-03-09 20:57:51  
images/Test_1000.jpg   238KB  2020-03-09 20:57:51  
images/Test_1012.jpg   174KB  2020-03-09 20:57:51  
images/Test_101.jpg    204KB  2020-03-09 20:57:51  
images/Test_1009.jpg   220KB  2020-03-09 20:57:51  
images/Test_1007.jpg   124KB  2020-03-09 20:57:51  
images/Test_1.jpg      294KB  2020-03-09 20:57:51  
images/Test_10.jpg     245KB  2020-03-09 20:57:51  
images/Test_1004.jpg   232KB  2020-03-09 20:57:51  
images/Test_0.jpg      227KB  2020-03-09 20:57:51  
images/Test_1006.jpg   207KB  2020-03-09 20:57:51  
images/Test_100.jpg    275KB  2020-03-09 20:57:51  
images/Test_1008.jpg   274KB  2020-03-09 20:57:51  
images/Test_1014.jpg   218KB  2020-03-09 20:57:51  
images/Test_1003.jpg   213KB  2020-03-09 20:57:51  
images/Test_1005.jpg   180KB  2020-03-09 20:57:51  
train.csv               33KB  2020-03-09 20:57:51  
test.csv                17KB  2020-03-09 20:57:51  
sample_submission.csv   52KB  2020-03-09 20:57:51  
```

**Comment by tanmaypandey7** [comment]
use this 
`pip install --upgrade --force-reinstall --no-deps kaggle`

**Comment by rbracco** [comment]
This failed for me. `pip install --upgrade --force-reinstall --no-deps kaggle` appeared to upgrade 1.5.4 to 1.5.6 but then `kaggle -v` would still show 1.5.4. Even a full uninstall, updating pip, and reinstalling failed. 

I was able to get it working by manually removing the kaggle binary (v 1.5.4) that was persisting at `/opt/anaconda/bin/kaggle`. and reinstalling. 

What worked for me
```
pip uninstall kaggle
pip install -U pip 
rm /opt/anaconda/bin/kaggle
pip install --upgrade --force-reinstall --no-deps kaggle
```

Notes:
1. `pip install -U pip` is for Linux/Mac, for windows use `python -m pip install -U pip`
2. For the final command you probably don't need `pip install --upgrade --force-reinstall --no-deps kaggle` but rather just `pip install kaggle` but I wanted to include the exact steps I used to get it working.
